### ISSUE TITLE
wrong about Eigen3

### ISSUE THREAD

**Issue by zhudaoruyi** [ppwwyyxx/OpenPano#89]
```
CMake Error at src/CMakeLists.txt:26 (add_executable):
  Target "image-stitching" links to target "Eigen3::Eigen" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at src/CMakeLists.txt:23 (add_library):
  Target "openpano" links to target "Eigen3::Eigen" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

**Comment by ppwwyyxx** [comment]
Please include necessary information with your issue:
1. What you did: the command you run / config you use.
2. What you observed: full logs, and other relevant information (e.g., images).
3. Your environment information.

**Comment by zhudaoruyi** [comment]
```bash
ubuntu@ubuntu:~/github/OpenPano$ sudo apt install build-essential sed cmake libjpeg-dev libeigen3-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
build-essential is already the newest version (12.1ubuntu2).
build-essential set to manually installed.
libjpeg-dev is already the newest version (8c-2ubuntu8).
sed is already the newest version (4.2.2-7).
libeigen3-dev is already the newest version (3.3~beta1-2).
cmake is already the newest version (3.5.1-1ubuntu3).
0 upgraded, 0 newly installed, 0 to remove and 367 not upgraded.
ubuntu@ubuntu:~/github/OpenPano$ mkdir build && cd build
ubuntu@ubuntu:~/github/OpenPano/build$ cmake ..
-- The CXX compiler identification is GNU 5.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test COMPILER_SUPPORTS_CXX11
-- Performing Test COMPILER_SUPPORTS_CXX11 - Success
-- Performing Test COMPILER_SUPPORTS_CXX0X
-- Performing Test COMPILER_SUPPORTS_CXX0X - Success
-- Try OpenMP CXX flag = [-fopenmp]
-- Performing Test OpenMP_FLAG_DETECTED
-- Performing Test OpenMP_FLAG_DETECTED - Success
-- Found OpenMP: -fopenmp  
-- Found JPEG: /usr/lib/aarch64-linux-gnu/libjpeg.so  
-- Found PythonLibs: /usr/lib/aarch64-linux-gnu/libpython3.5m.so (found version "3.5.2") 
-- Configuring done
CMake Error at src/CMakeLists.txt:26 (add_executable):
  Target "image-stitching" links to target "Eigen3::Eigen" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at src/CMakeLists.txt:23 (add_library):
  Target "openpano" links to target "Eigen3::Eigen" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


-- Generating done
-- Build files have been written to: /home/ubuntu/github/OpenPano/build
ubuntu@ubuntu:~$ whereis eigen3
eigen3: /usr/include/eigen3
```

**Comment by zhudaoruyi** [comment]
[https://stackoverflow.com/questions/53991812/cmake-find-package-not-working-for-eigen](https://stackoverflow.com/questions/53991812/cmake-find-package-not-working-for-eigen)

**Comment by ppwwyyxx** [comment]
Thanks for finding it out!

Turned out that this `Eigen3::Eigen` target was introduced since eigen 3.3.1: http://eigen.tuxfamily.org/index.php?title=ChangeLog#Eigen_3.3.1

Closed by #90
